app/components/obs/inputs/ObsInput.ts

```diff
@@ -1,4 +1,3 @@
-import Vue from 'vue';
```

Comment:
why is the Vue import removed here?

Reply:
It wasn't being called anywhere, but restored